department-of-veterans-affairs/caseflow
# JonathanH/appeals-13250

Resolves [13250](https://vajira.max.gov/browse/APPEALS-13250?workflowName=VA+Caseflow+Sandbox+Story+Enabler+Workflow&stepId=20)

### Description

- Added migration for cavc_selection_basis to  Caseflow DB
- Added cavc_selection_basis model
- Added seed file and script to add selection basis to DB
- Updated Cavc Dashboard component to load selection basis data

### Acceptance Criteria

### Testing Plan
Verify selection_basis loads on the CAVC Dashboard using Redux Dev Tools:

1. In demo environment, go to Search and search for a CAVC Remand by veteran file number (200000250 - 200000260 should have CAVC Remands)
2. Go to case details page for that appeal
3. Open Redux Dev Tools
4. Go to State tab
5. Expand the cavcDashboard section and verify no data exists for the selection_basis key
6. From the case details page, add "/cavc_dashboard" to the URL and go to that page
7. In the Redux Dev Tools, verify that the cavcDashboard section selection_basis key has objects with the values from the linked Jira story

### Code Documentation Updates

### Storybook Story
*For Frontend (Presentationa) Components*

### Database Changes
*Only for Schema Changes*
